innovation.game.php

```diff
@@ -13686,7 +13690,24 @@ function stPlayerInvolvedTurn() {
                 self::executeDraw($player_id, 7, 'score');
                 self::executeDraw($player_id, 7, 'hand');
                 break;
-
+            
+            // id 413, Echoes age 8: Crossword
+            case "413N1":
```

Comment:
You could draw a card automatically if only one bonus present (presumably the one on Crossword)